tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/graphql/entity/TaskDTO.java

```diff
@@ -33,37 +37,41 @@
 
 public final class TaskDTO {
 
-  private String id;
-  private String processInstanceId;
+  @GraphQLField @GraphQLNonNull private String id;
+  @GraphQLField private String processInstanceId;
 
   /** Field is used to resolve task name. */
   private String flowNodeBpmnId;
 
   private String flowNodeInstanceId;
 
   /** Field is used to resolve process name. */
-  private String processDefinitionId;
+  @GraphQLField private String processDefinitionId;
 
   /** Fallback value for process name. */
   private String bpmnProcessId;
 
-  private String creationTime;
-  private String completionTime;
-  private String assignee;
-  private String[] candidateGroups;
-  private String[] candidateUsers;
-  private TaskState taskState;
-  private String[] sortValues;
-  private boolean isFirst = false;
-  private String formKey;
+  @GraphQLField @GraphQLNonNull private String creationTime;
+  @GraphQLField private String completionTime;
+  @GraphQLField private String assignee;
+  @GraphQLField private String[] candidateGroups;
+  @GraphQLField private String[] candidateUsers;
+  @GraphQLField @GraphQLNonNull private TaskState taskState;
```

Comment:
Does NonNull also validate the case of Blank string?

Reply:
In our [Schama](https://github.com/camunda/zeebe/blob/main/tasklist/webapp/src/main/resources/graphql/tasklist.graphqls), `String!` type indicates a non-nullable string, but blank values are accepted